src/crowdstrike/foundry/function/_internal/cslogger.py

```diff
@@ -5,6 +5,8 @@
     Any,
 )
 
+LOG_LEVELS = {'DEBUG', 'INFO', 'WARN', 'WARNING', 'ERROR', 'CRITICAL'}
```

Comment:
is this for the internal impl details only? or is this something users will be tieing into?